src/plugins/gauge/components/GaugeFormController.vue

```diff
@@ -118,21 +121,30 @@ export default {
             min: this.model.value.min
         };
     },
+    mounted() {
+        this.openmct.forms.on('formPropertiesChanged', this.onFormPropertyChange);
+    },
+    beforeDestroy() {
+        this.openmct.forms.off('formPropertiesChanged', this.onFormPropertyChange);
+    },
     methods: {
+        onFormPropertyChange(data) {
+            this.changes = data?.configuration?.gaugeController || {};
+        },
         onChange(event) {
             const data = {
                 model: this.model,
                 value: {
-                    gaugeType: this.model.value.gaugeType,
-                    isDisplayMinMax: this.isDisplayMinMax,
-                    isDisplayCurVal: this.isDisplayCurVal,
-                    isDisplayUnits: this.isDisplayUnits,
+                    gaugeType: this.changes.gaugeType || this.model.value.gaugeType,
+                    isDisplayMinMax: this.changes.isDisplayMinMax === undefined ? this.isDisplayMinMax : this.changes.isDisplayMinMax,
```

Comment:
I prefer using `=== undefined` here, if that's OK?

Reply:
On second glance, in this case `=== undefined` and using `??` are not completely functionally the same. `??` will assign the left-hand side value if it is not `undefined` OR `null`, otherwise it will assign the right-hand side value.

So if `null` is a valid value here, then we shouldn't use `??`.

If `null` is not a valid value, then it's all syntactic sugar and entirely up to your preference / whatever you find more readable. :)